config/initializers/inflections.rb

```diff
@@ -11,6 +11,8 @@
 # end
 
 # These inflection rules are supported but not enabled by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
-# end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym 'UUID'
+  inflect.acronym 'HTML'
+  inflect.acronym 'HTTP'
+end
```

Comment:
Had to add this to get Zeitwerk to like our files.

Reply:
Oh look... you're using it ... Could add `API` I think.

Disregard above comments :)